addons/point_of_sale/static/src/js/models.js

```diff
@@ -1794,6 +1864,8 @@ export class Orderline extends PosModel {
         this.pos = options.pos;
         this.order = options.order;
         this.price_manually_set = options.price_manually_set || false;
+        this.uuid = this.uuid || uuidv4();
```

Comment:
this is the exact same line in pos restaurant but there is one more condition on pos_restaurant. Looks strange...